lib/Jit/LLILCJit.cpp

```diff
@@ -641,3 +641,12 @@ void ObjectLoadListener::getRelocationTypeAndAddend(uint8_t *FixupAddress,
     llvm_unreachable("Unknown reloc type.");
   }
 }
+
+unsigned LLILCJit::getMaxIntrinsicSIMDVectorLength(DWORD CpuCompileFlags) {
+  return GenIR::getPreferIntrinsicSIMDVectorLength();
+}
+
+bool LLILCJitContext::isAltJit() {
+  JitOptions JitOptions(*this);
+  return JitOptions.IsAltJit;
```

Comment:
JitContext->Options has type ::Options.
We need JitOptions here.
I think they are different structs.